src/components/SocialListItem.tsx

```diff
@@ -1,42 +1,39 @@
 // Libraries
 import React from "react"
-import styled from "@emotion/styled"
+import { Flex, Box, chakra } from "@chakra-ui/react"
 
 // Components
 import Icon from "./Icon"
 
 // Styles
-const Item = styled.div`
-  display: flex;
-  width: 100%;
-  padding: 0.5rem 0;
-  align-items: center;
-`
-
-const StyledIcon = styled(Icon)`
-  padding-right: 0.75rem;
-`
-
-const ChildrenContainer = styled.div`
-  font-style: italic;
-
-  > a {
-    font-style: normal;
-  }
-`
+const StyledIcon = chakra(Icon)
 
 export interface IProps {
   children?: React.ReactNode
   socialIcon: string
 }
 const SocialListItem: React.FC<IProps> = ({ children, socialIcon }) => {
   return (
-    <Item>
+    <Flex w="100%" py="0.5rem" px="0" alignItems="center">
       <div>
-        <StyledIcon name={socialIcon} size={"2.5rem"} color={true} />
+        <StyledIcon
+          pr={"0.75em"}
```

Comment:
Lets try to use as much Chakra values as possible. You can see the spacing values [here](https://chakra-ui.com/docs/styled-system/theme#spacing).
```suggestion
          pr={3}
```